src/rustls.h

```diff
@@ -674,26 +675,12 @@ typedef uint32_t (*rustls_session_store_put_callback)(rustls_session_store_userd
                                                       const struct rustls_slice_bytes *key,
                                                       const struct rustls_slice_bytes *val);
 
-/**
- * Rustls' list of supported protocol versions. The length of the array is
- * given by `RUSTLS_ALL_VERSIONS_LEN`.
- */
```

Comment:
Hm, I think I must have dropped these comments by mistake in my 0.14 prep branch and not re-run `cbindgen` pre-merge. Will fix.

Reply:
> Hm, I think I must have dropped these comments by mistake in my 0.14 prep branch 

Nevermind, just using the wrong `cbindgen` version on this computer. The comments in `src/enum.rs` are still there.